src/JwtClaimsServiceProvider.php

```diff
@@ -32,7 +33,7 @@ public function makeAuthorizationServer()
             $this->app->make(AccessTokenRepository::class), // BenBjurstrom\JwtClaims\AccessTokenRepository
             $this->app->make(ScopeRepository::class),
             new CryptKey( 'file://'.Passport::keyPath('oauth-private.key'), null, false ),
-            new CryptKey('file://'.Passport::keyPath('oauth-public.key'), null, false )
+            app('encrypter')->getKey()
```

Comment:
@benbjurstrom  FYI, my original commit here was wrong (this second argument is expected to be a secret encryption key as a string, rather than a CryptKey instance.  

However looking at the upstream they use ` app('encrypter')->getKey()` here rather than the public key and as far as I can see this key is supposed to be a private secret string unique to the app rather than a filename